addons/discuss-button/addon.json

```diff
@@ -34,5 +34,5 @@
   ],
   "versionAdded": "1.0.0",
   "tags": ["community", "forums"],
-  "enabledByDefault": false
+  "enabledByDefault": true
```

Comment:
normally stuff are false rather then true. This should probably be false

Reply:
Addons that are normally enabled by default have to go through approval by @WorldLanguages I think..